packages/popover/__tests__/popover.spec.ts

```diff
@@ -0,0 +1,231 @@
+import { mount } from '@vue/test-utils'
+import Popover from '../src/index.vue'
+import * as popper from '@popperjs/core'
+import { h, ref } from 'vue'
+import { triggerEvent } from '@element-plus/utils/aria'
+import directive from '../src/directive'
+
+const AXIOM = 'Rem is the best girl'
+
+const popperMock = jest.spyOn(popper, 'createPopper').mockImplementation(() => ({
+  update: jest.fn(),
+  forceUpdate: jest.fn(),
+  setOptions: jest.fn(),
+  destroy: jest.fn(),
+  state: null,
+}))
+
+describe('Popover.vue', () => {
+  afterAll(() => {
+    popperMock.mockReset()
+  })
+
+  beforeEach(() => {
+    popperMock.mockClear()
+    jest.useFakeTimers()
+  })
+
+  test('render', () => {
+    const wrapper = mount(Popover, {
+      props: {
+        trigger: 'click',
+      },
+      slots: {
+        reference: h('div', AXIOM),
```

Comment:
It will be better if you guys use `JSX`  instead of h or createVNode function

Reply:
usually this is recommended by returning a function instead of `VNode` for slots
And also, for simple testing, I do not think that introducing another parser here for test files to parse `jsx` is a good idea, you see. The node structure is quite simple, and also `createVNode` function requires extra parameters which is not recommended for user to use it directly, `h` function handles it for you. ;)

Reply:
As you said here we did not implement `jsx` in this repo due to some legacy issue (we are using customized storybook for developing) and we plan to switch it to the old fashioned way in near future. Thanks for bringing this up btw.